concordium-std/src/lib.rs

```diff
@@ -275,7 +376,9 @@ pub mod constants;
 mod impls;
 pub mod prims;
 mod traits;
-mod types;
+// TODO: For reviewers: I made this pub to avoid a never used warning for
+// `ChainMetadata`. I am not sure why `pub use types::*;` isn't sufficient.
+pub mod types;
```

Comment:
There is a type called `ChainMetadata` in contracts_common, which is also re-exported from here.

Maybe we should have some other name for the type alias.

I don't think we want to export the `types` module.

Reply:
Ah, that's why. I've called it `ChainMeta` now.